t/proto_template/README.md

```diff
@@ -0,0 +1,57 @@
+# Context
+
+You must have various tools installed in order to generate protos. The tool does not distinguish between language types and instead seeks to generate protos for a variety of supported languages. Currently only go is supported. Node will be supported in the near future.
+
+## Tools
+
+Install `protoc`, a compiler for generating language specific protos based on .proto files. 
+
+```
+  brew install protobuf # for MacOS users
+  apt install -y protobuf-compiler # for Linux users
```

Comment:
Nit: apt is only for debian-based distros, other linux distros have other package managers (dnf/yum/pacman)